src/TypeScript/DefinitionWriter.cpp

```diff
@@ -106,7 +106,8 @@ void DefinitionWriter::visit(InterfaceMeta* meta)
         }
     }
 
-    _buffer << std::endl << "\tdeclare class " << meta->jsName << getTypeParametersStringOrEmpty(clang::cast<clang::ObjCInterfaceDecl>(meta->declaration));
+    _buffer << std::endl
+            << _docSet.getCommentFor(meta).toString("\t") << "\tdeclare class " << getTypeParametersStringOrEmpty(clang::cast<clang::ObjCInterfaceDecl>(meta->declaration));
```

Comment:
I think that the class name is missing here.

Here is the declaration for `NSObject`:

```typescript
declare class  implements NSObjectProtocol {
    // ...
}
```